crates/common/src/schema_response.rs

```diff
@@ -205,6 +213,8 @@ pub fn schema_response(
         collections: vec![],
         functions,
         procedures,
+        capabilities: None,
+        request_arguments: None,
```

Comment:
We should be advertising the `headers` request argument in here, the plugin just YOLOs the stuff it sends but once we supports sending request arguments directly from engine we'll want to use this to check we're sending the correct types etc.

Reply:
With other nullable arguments, engine sends null if it doesn't have a value, I imagine it'll do the same here. So you're fine to add it so long as it's nullable.

Reply:
Looks great, thanks!